### PR TITLE
Use swagger api for Site Replication Status

### PR DESCRIPTION
**Pull request by cesnietor**
### Test Steps

1. Go to /settings/site-replication
2. Add a site to replicate data
3. Click on `Replication Status`
4. It should load the data from tha replication status

### CODE REVIEW

**Review by bexsoft — COMMENTED** [review]
Please review failing tests

File: web-app/src/screens/Console/Configurations/SiteReplication/SiteReplicationStatus.tsx [hunk 1]
```diff
@@ -113,8 +111,24 @@ const SiteReplicationStatus = () => {
...
+        buckets: true,
+        groups: true,
+        policies: true,
+        users: true,
+      })
+      .then((res: HttpResponse<SiteReplicationStatusResponse, ApiError>) => {
+        setStats(res.data);
+      })
+      .catch(
+        async (res: HttpResponse<SiteReplicationStatusResponse, ApiError>) => {
```
Comment (prakashsvmx) [inline web-app/src/screens/Console/Configurations/SiteReplication/SiteReplicationStatus.tsx:126]: `async` can be omitted  as there is no `await` in the block

**Review by prakashsvmx — APPROVED** [review]
Verified. LGTM :+1:

**Comment by cesnietor** [comment]
> Please review failing tests

@bexsoft  these seem unrelated to this change.

File: web-app/src/screens/Console/Configurations/SiteReplication/SiteReplicationStatus.tsx (hunk 1 above)
Comment (cesnietor) [reply web-app/src/screens/Console/Configurations/SiteReplication/SiteReplicationStatus.tsx]: done, lost your approval :(

Comment (cesnietor) [reply web-app/src/screens/Console/Configurations/SiteReplication/SiteReplicationStatus.tsx]: I removed it also from other places I missed.

**Review by reivaj05 — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
Remaining failure not related to this change, there were a couple of failures in the first iteration of this PR. hence LGTM

**Pull request merged**